docs/source/en/quicktour.md

```diff
@@ -23,18 +23,20 @@ Get up and running with 🤗 Transformers! Whether you're a developer or an ever
 Before you begin, make sure you have all the necessary libraries installed:
 
 ```bash
-!pip install transformers datasets
+pip install transformers datasets
```

Comment:
The exclamation mark needs to stay for the conversion of this doc page into notebook.

Reply:
done